integration/data/test_backup.py

```diff
@@ -4,16 +4,16 @@
 
 import cmd
 import common
-from common import dev, backing_dev  # NOQA
-from common import read_dev, read_from_backing_file, BACKUP_DEST
+from common import replica1, replica2, controller, backing_replica1, backing_replica2  # NOQA
+from common import read_dev, read_from_backing_file
 from snapshot_tree import snapshot_tree_build, snapshot_tree_verify_backup_node
 
 VOLUME_NAME = 'test-volume_1.0'
 VOLUME_SIZE = str(4 * 1024 * 1024)  # 4M
 BLOCK_SIZE = str(2 * 1024 * 1024)  # 2M
 
 
-def test_backup(dev):  # NOQA
+def backup_test(dev, backupTarget):  # NOQA
```

Comment:
python uses `_` to separate words. So `backup_target` instead of `backupTarget`.